inventory-generation/identity-management/main.yml

```diff
@@ -30,27 +34,32 @@
         - "ocp-admin-credentials.json"
         - "ocp-ldap-sa-credentials.json"
 
-    - name: "Add users to inventory"
-      set_fact:
-        users: "{{ (users | default([])) + [ { 'first_name': (item.first_name | trim), 'last_name': (item.last_name | trim), 'email': (item.email | trim) , 'user_name': (item.email.split('@')[0] | trim) } ] }}"
+    - include_tasks: process_user.yml
+      loop_control:
+        loop_var: user_item
       loop: "{{ engagement_users }}"
 
-    - name: "Add LDAP Service Account"
+    - include_tasks: process_groups.yml
+      loop_control:
+        loop_var: group_item
+      loop: "{{ user_options.user_roles.options }}"
```

Comment:
need some default values defined in case the `user_options` and/or `user_roles` isn't defined

Reply:
These values are different for `residency` and `do500` engagements and I am not sure if we need to hardcore it. Probably it will be better to fix lack of the `runtime.json` file in the engagement repository.